code/modules/mining/machine_redemption.dm

```diff
@@ -250,6 +251,9 @@
 	var/datum/component/material_container/mat_container = materials.mat_container
 	switch(action)
 		if("Claim")
+			if(!materials.silo)
+				to_chat(usr, span_warning("Mining points undistributable while unsynced to an ore silo."))
```

Comment:
You need mining points to set up RnD as a golem.
You need RnD as a golem to create an ore silo.